### PR TITLE
[#4077] improvement(docs): Fixed an incorrect description and some incomplete examples

### PR DESCRIPTION
**Pull request by JinsYin**
### What changes were proposed in this pull request?

Fixed an incorrect description.

### Why are the changes needed?

Fix: #4145

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

No testing required

### CODE REVIEW

**Comment by jerryshao** [comment]
@yuqi1129 @diqiu50 please help to review this.

File: docs/trino-connector/supported-catalog.md [hunk 1]
```diff
@@ -14,7 +14,6 @@ The catalogs currently supported by the Apache Gravitino connector are as follow
 
 ## Create catalog
 
-Trino itself does not support creating catalogs.
```
Comment (yuqi1129) [inline docs/trino-connector/supported-catalog.md:17]: @diqiu50
Please help to clarify it, thanks.

Comment (diqiu50) [reply docs/trino-connector/supported-catalog.md:17]: In the current minimum support version and required, removing this sentence is ok.

**Review by diqiu50 — APPROVED** [review]
LGTM

**Review by diqiu50 — APPROVED** [review]

**Review by jerryshao — APPROVED** [review]

**Pull request merged**